lib/src/Uncertainty/Algorithm/Simulation/PhysicalSpaceCrossEntropyImportanceSampling.cxx

```diff
@@ -59,6 +59,10 @@ PhysicalSpaceCrossEntropyImportanceSampling::PhysicalSpaceCrossEntropyImportance
     activeParameters_ = activeParameters;
   
     Point parameters(auxiliaryDistribution_.getParameter());
+    
+    if (activeParameters_.getSize() != initialAuxiliaryDistributionParameters.getSize())
+    throw InvalidArgumentException(HERE) << "In PhysicalSpaceCrossEntropyImportanceSampling::PhysicalSpaceCrossEntropyImportanceSampling, input sample size (" << activeParameters_.getSize() << ") does not match initial auxiliary distribution parameters size (" << initialAuxiliaryDistributionParameters.getSize() << ").";
+    
```

Comment:
that's the spirit, now you must also check that these two match the distribution parameter size, else it can still crash

Reply:
Actually if we do not optimize all the parameters of the auxiliary distribution, active parameters and initialAuxiliaryDistributionParameters could have a size < distribution parameters.

Reply:
yes, we must check that it is <=

Reply:
You're true. This is fixed